resources/roadmap/index.md

```diff
@@ -221,16 +235,13 @@ meta_descr: meta_descr.roadmap
                         <h2>{% t roadmap.future %}</h2>
                             <ul>
                                 <h3 class="months">{% t roadmap.comingsoon %}</h3>
-                                    <li class="ongoing"><a href="https://haveno.exchange">{% t roadmap.haveno %}</a></li>
-                                    <li class="ongoing"><a href="https://github.com/seraphis-migration/strategy/wiki">{% t roadmap.seraphis-jamtis %}</a></li>
                                     <li class="ongoing"><a href="https://github.com/Rucknium/OSPEAD">{% t roadmap.ospead %}</a></li>
-                                    <li class="ongoing">{% t roadmap.ringsize-sera-tryp %}</li>
-                                    <li class="ongoing">{% t roadmap.tryptych %}</li>
                                     <li class="ongoing">{% t roadmap.xmr-eth-atomicswaps %}</li>
                                     <li class="ongoing">{% t roadmap.bp++ %}</li>
-                                    <li class="ongoing">{% t roadmap.lws-subaddress %}</li>
-                                    <li class="upcoming">{% t roadmap.returnaddr %}</li>
-                                    <li class="upcoming">{% t roadmap.layer2 %}</li>
+                                    <li class="ongoing">{% t roadmap.cuprate %}</li>
+                                    <li class="upcoming"><a href="https://github.com/seraphis-migration/strategy/wiki">{% t roadmap.seraphis-jamtis %}</a></li>
```

Comment:
```suggestion
                                    <li class="upcoming"><a href="https://github.com/seraphis-migration/strategy/wiki/Our-Wallet-Rewrite-Project-After-Priority-Switched-From-Seraphis-to-FCMPs">{% t roadmap.seraphis-jamtis %}</a></li>
```
rbrunner has added a new entry to the seraphis wiki above. it explains the "Seraphis & Jamtis" -> "Seraphis codebase & Jamtis" shift.

Reply:
I believe we shouldn't link the FCMP for seraphis roadmap, instead we should link for FCMP roadmap itself